web/regression/javascript/file_manager/FileManager.spec.js

```diff
@@ -162,7 +162,7 @@ describe('FileManger', ()=>{
 
       await user.click(ctrl.container.querySelector('[name="menu-shared-storage"]'));
       await user.click(ctrl.container.querySelector('[data-label="Shared Storage"]'));
-      expect(ctrl.container.querySelector('button[aria-label="Shared Storage"]')).not.toBeNull();
+      expect(ctrl.container.querySelector('button[aria-label="Shared Storage"]')).toBeNull();
```

Comment:
This is not the right way. aria-label should be added button for accessibility reasons.